src/components/login/Login.js

```diff
@@ -81,17 +95,22 @@ class Login extends React.Component {
    */
   async login() {
     try {
+        /*
       const requestBody = JSON.stringify({
         username: this.state.username,
-        name: this.state.name
+        password: this.state.password
       });
-      const response = await api.post('/users', requestBody);
+         */
+      const requestHeader = ArrayBuffer.from(`${this.state.username}, ${this.state.password}`, 'utf8').toString('base64');
 
-      // Get the returned user and update a new object.
-      const user = new User(response.data);
+      //Buffer.from(`${this.state.username}, ${this.state.password}`, 'utf8').toString('base64');
+      const response = await api.get('/user/login', {headers: {'Authorization': `${requestHeader}`}});
```

Comment:
There is no need to put all your variables into this kind of explicit string declaration. The requestHeader here is guaranteed to be a string, so just use `requestHeader` and not `${requestHeader}`.